src/tests/test_day2.py

```diff
@@ -26,18 +39,299 @@ def test_deploy_day2_nodes_cloud(self, cluster, new_cluster_configuration: Clust
             cluster.prepare_for_installation()
             cluster.start_install_and_wait_for_installed()
 
-        # TODO: Use a proper structure instead of mimicking cli options
-        args = SimpleNamespace()
-        args.api_client = cluster.api_client
-        args.pull_secret = new_cluster_configuration.pull_secret
-        args.with_static_network_config = new_cluster_configuration.is_static_ip
-        args.num_day2_workers = global_variables.num_day2_workers
-        args.ssh_key = global_variables.ssh_public_key
-        args.install_cluster = True
-
-        if new_cluster_configuration.proxy:
-            args.http_proxy = new_cluster_configuration.proxy.http_proxy
-            args.https_proxy = new_cluster_configuration.proxy.https_proxy
-            args.no_proxy = new_cluster_configuration.proxy.no_proxy
-
-        day2.execute_day2_cloud_flow(new_cluster_configuration.cluster_id, args, new_cluster_configuration.is_ipv6)
+        self.execute_day2_flow(
+            cluster.api_client,
+            new_cluster_configuration.proxy,
+            new_cluster_configuration.cluster_id,
+            new_cluster_configuration.pull_secret,
+            global_variables.ssh_public_key,
+            global_variables.num_day2_workers,
+            new_cluster_configuration.is_static_ip,
+            new_cluster_configuration.is_ipv6,
+            True,
+        )
+
+    @classmethod
```

Comment:
```suggestion
    @classmethod
```

just wondering about all those ``classmethod``s, do we gain anything from having a class that only has class methods, or maybe we can just transform to the less-meta use of just instance methods
In my opinion, we should choose the latter for simplicity and because pytest will eventually instantiate those classes

Reply:
yeah not much benefit, removed